packages/core/docs/commercetools/api-client.md

```diff
@@ -71,6 +71,50 @@ export interface TokenEvents {
   onTokenRemove?: () => void;
 }
 ```
+
+### Token handling
+
+Commercetools platform requires token to load data from the API. Our api-client generates this token itself, but you have to store it to prevent generating this multiple times. In order to do it, you can use  `onTokenChange` event to react on token updating and `setup` function for reconfiguration.
+
+The most common use-case is storing that token using cookies:
+
+```js
+import { setup } from '@vue-storefront/commercetools-api';
+import { config } from './your/config';
+
+const currentToken = app.$cookies.get('ct-token');
+
+const onTokenChange = (token) => {
+  app.$cookies.set('ct-token', token);
```

Comment:
how backend knows that it's a right token, does the cookie always need to be named `ct-token` ?

Reply:
You can name it whatever you like, it's just cookie name that stores the token. Backend doesn't have any validation stuff, if token is invalid the API will return an error, but there is no way to put invalid token there unless you do it manually or there is a bug in the code